editoast/src/models/paced_train.rs

```diff
@@ -19,7 +19,7 @@ use super::Tags;
 use crate::models::prelude::*;
 use crate::models::train_schedule::TrainSchedule;
 
-#[derive(Debug, Clone, Model)]
+#[derive(Debug, Default, Clone, Model)]
```

Comment:
nit: same about cfg_attr

Reply:
I chose not too because the corresponding `TrainSchedule` doesn't have it.

Reply:
OK, I tested to move all of them as `#cfg_attr(test, derive(Default))]` and it seems fine. I feel it's probably a better thing because I'm not sure it make sense to have a `Default` implementations for these (except, it's handy for tests). Changed in 6ec2e277.